src/About/Feedback.tsx

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
+ */
+
+import React, { useMemo, useState } from 'react';
+
+import { DialogButton, GenericDialog } from '../Dialog/Dialog';
+import Dropdown, { DropdownItem } from '../Dropdown/Dropdown';
+import logger from '../logging';
+import describeError from '../logging/describeError';
+import AboutButton from './AboutButton';
+import sendFeedback from './sendFeedback';
+
+export interface FeedbackPaneProps {
+    isVisible: boolean;
+    onHide: () => void;
+    categories?: string[];
+}
+
+const FeedbackDialog = ({
+    categories,
+    onHide,
+    isVisible,
+}: FeedbackPaneProps) => {
+    const [feedback, setFeedback] = useState('');
+    const [sayThankYou, setSayThankYou] = useState(false);
+    const [sendingFeedback, setSendingFeedback] = useState(false);
+
+    const categoryItems = useMemo(() => {
+        if (!categories?.length) return undefined;
+
+        const items = ['Select a category', ...categories].map(category => ({
+            label: category,
+            value: category,
+        }));
+        return items;
+    }, [categories]);
+
+    const [selectedCategory, setSelectedCategory] = useState<
+        DropdownItem | undefined
+    >(categoryItems ? categoryItems[0] : undefined);
+
+    return (
+        <GenericDialog
+            isVisible={isVisible}
+            onHide={onHide}
+            closeOnUnfocus
+            title="Feedback"
+            showSpinner={sendingFeedback}
+            footer={
+                <>
+                    {sayThankYou ? (
+                        <DialogButton
+                            variant="secondary"
+                            onClick={() => {
+                                setSayThankYou(false);
+                                setFeedback('');
+                            }}
+                        >
+                            Give more feedback
```

Comment:
What is the purpose of this button?

Reply:
It returns to the previous screen (with the feedback form)
so that the user can give more feedback.

I guess before this was an actual necessity to make giving feedback multiple times possible.
So this could be removed if necessary. I guess that decision would fall onto UX though so I'll ask.

Reply:
To me, it reads as either that (additional feedback) or "more details about what I just wrote". What's the point of having this if the user will probably just dump all his ideas into one form?